public/team/index.html

```diff
@@ -75,38 +75,44 @@
 <article>
 <h1>The Team</h1>
+<p>
+Invidious is a community maintained project, this page lists the main people who work/worked on it.
+</p>
+
 <h2>Project managers:</h2>
 <p>
-<a href="https://github.com/TheFrenchGhosty">TheFrenchGhosty</a>: Creator of this website - Finances manager - Documentation manager
+<a href="https://github.com/TheFrenchGhosty">TheFrenchGhosty</a> (2019-Today): Project manager - Finances manager - Documentation manager - Creator of this website - System administrator
 <br>
-<a href="https://github.com/Perflyst">Perflyst</a>: Infrastructure manager
+<a href="https://github.com/Perflyst">Perflyst</a> (2019-Today): Project manager - Infrastructure manager - System administrator
 </p>
-<p class="text-muted">Both project managers have full access to everything related to the project (making the project <a href="https://en.wikipedia.org/wiki/Bus_factor">bus factor</a> resistant).</p>
+<p class="text-muted">Both project managers have full access to everything related to the project (including the cryptocurrency wallets) (making the project <a href="https://en.wikipedia.org/wiki/Bus_factor">bus factor</a> resistant).</p>
 <h2>Developers:</h2>
 <p>
-<a href="https://github.com/SamantazFox">Samantaz Fox</a>: Main developer - Main tester - Administrator of the test instance
+<a href="https://github.com/SamantazFox">Samantaz Fox</a> (2021-Today): Lead developer - Main tester - Administrator of the test instance
+<br>
+<a href="https://github.com/syeopite">syeopite</a> (2021, 2023-Today): Core developer
 <br>
-<a href="https://github.com/unixfox">unixfox</a> Secondary developer - Kubernetes manager - Documentation manager
+<a href="https://github.com/unixfox">unixfox</a> (2020-Today): Core developer - Kubernetes manager - Documentation manager
 </p>
+<br>
 <h1>Main contributors</h1>
 <p>
-<a href="https://github.com/matthewmcgarvey">Matthew McGarvey</a>: Development
+<a href="https://github.com/ChunkyProgrammer">ChunkyProgrammer</a>: Development
 <br>
-<a href="https://github.com/TiA4f8R">TiA4f8R</a>: Reverse engineering - Research
+<a href="https://github.com/TeamNewPipe">The NewPipe team</a>, mainly <a href="https://github.com/AudricV">AudricV</a>: Reverse engineering - Research
 <br>
-<a href="https://github.com/TeamNewPipe">The NewPipe team</a>: Reverse engineering - Research
+<a href="https://github.com/FreeTubeApp">The FreeTube team</a>, mainly <a href="https://github.com/absidue">Absidue</a>: Reverse engineering - Research
 <br>
 <a href="https://github.com/yt-dlp">The yt-dlp team</a>: Reverse engineering - Research
 </p>
-<br>
 <h1>Past contributors</h1>
 <p><a href="https://github.com/omarroth">Omar Roth</a>: Original developer
 <br>
-<a href="https://github.com/syeopite">syeopite</a>: Was the second main developer in the team, until we lost complete contact with him
-<br>
-<a href="https://github.com/saltycrys">saltycrys</a>: Development
+<a href="https://github.com/saltycrys">saltycrys</a>: Major development
```

Comment:
I wanted to keep "develop..." to "follow" the fact that you are lead/core "developer", but yeah, it's not great.

I'll use your suggestion since I can't really think of a better way to say "major development" that keeps "develop..."